commands/rdp.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import taskcluster
+import subprocess
+
+
+def main(task_id_or_url, *extra_client_args):
+    prefix = "https://community-tc.services.mozilla.com/tasks/"
+    if task_id_or_url.startswith(prefix):
+        task_id = task_id_or_url[len(prefix):]
+    else:
+        task_id = task_id_or_url
+
+    options = taskcluster.optionsFromEnvironment()
+    queue = taskcluster.Queue(options)
+    rdp_info = queue.getLatestArtifact(task_id, "project/servo/rdp-info")
+
+    supported_clients = [
+        ["xfreerdp", "/v:{host}:{port}", "/u:{username}", "/p:{password}"],
```

Comment:
@paulrouget What RDP client do you use on mac? If it has CLI interface we could add it here.

Reply:
You could also create a .rdp file: https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/rdp-files

```
auto connect:i:1
full address:s:{host}:{port}
username:s:Administrator
```

Not sure how the password can be stored.

Reply:
Do you feel adding a script for this is worth it, if you still need to then manually open that file and copy-paste a password? Either way this seems like a different script, so let’s land this and we can always add something else later.